### ISSUE TITLE
HTTP Cache API: JS SDK

### ISSUE THREAD

**Issue by guybedford** [fastly/js-compute-runtime#991]
This is to track the JS SDK additions in Q4 to support HTTP Cache API

**Comment by abharadwaj999** [comment]
Can we have this by EoY (Q4) as we do have JS customers waiting on HTTP Cache API support ?

**Comment by guybedford** [comment]
Hostcall parity is exactly the Q4 goal. I haven't fully analyzed the surface area of the new HTTP Cache API yet, and KVStore v2 is next on the list, but it seems fairly likely we should be able to complete this in Q4.

**Comment by abharadwaj999** [comment]
apologies, but just for my understanding, does "hostcall parity" equate to "JS SDK availability in guest for HTTP Cache API " or something else host specific only?

**Comment by guybedford** [comment]
Yes we want to fully support the latest host APIs in the JS SDK.
